### ISSUE TITLE
Get data over Bluetooth (for example, from Accu-Check Guide device) 

### ISSUE THREAD

**Issue by Athesis** [Faltenreich/Diaguard#53]
It would be an awesome feature if it's possible to get the recorded data from an Accu-Check device.

**Comment by Faltenreich** [comment]
I agree that this would be awesome, but this feature would introduce privacy questions, legal topics and require continuous adjustements to keep up with the APIs of ACCU-CHECK. Unfortauntely these are expenses I cannot handle as an independent developer and therefore I have to reject this request. I hope you understand.
